apps/cyberstorm-nextjs/next.config.js

```diff
@@ -1,6 +1,9 @@
+const withPreconstruct = require("@preconstruct/next");
+
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  transpilePackages: ["react"],
```

Comment:
Is this line actually necessary?

Reply:
Alternative take - should it include `@thunderstore/cyberstorm`? (I just read [this note](https://nextjs.org/blog/next-13-1#built-in-module-transpilation-stable) quickly, didn't dig any deeper, so this is completely unloaded question.)

Reply:
I'm assuming preconstruct takes care of that with the `withPreconstruct` wrapper, and that this line was a leftover from when @Oksamies was manually trying to declare the packages here, thus the comment

Reply:
Well it seems that I've added [this line](https://github.com/thunderstore-io/thunderstore-ui/blob/master/apps/nextjs/next.config.js#L2) when making the cyberstorm components work with the old nextjs app, and I assume transpilePackages does somewhat the same thing. But I want to strong emphasise that this is all assumptions.

Reply:
Yeah I'm just asking if @Oksamies tried to run the project without this config line, since it seems like what he originally tried to do with it didn't end up being necessary after all (based on Discord screenshots).

Reply:
Ye, I tried running it without the line. It gave an weird React related error. That error lead me to an github thread, which listed this as an solution.